src/AppBundle/Search/SearchResultsProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AppBundle\Search;
+
+use AppBundle\Entity\Committee;
+use AppBundle\Repository\CommitteeMembershipRepository;
+use AppBundle\Repository\CommitteeRepository;
+
+class SearchResultsProvider
+{
+    private $committee;
+    private $membership;
+
+    public function __construct(CommitteeRepository $committee, CommitteeMembershipRepository $membership)
+    {
+        $this->committee = $committee;
+        $this->membership = $membership;
+    }
+
+    public function findCommittees()
+    {
+        // the following part is a fixture only, for this issue
+
+        /** @var Committee[] $committees */
+        $committees = $this->committee->findBy([], ['name' => 'asc'], 20);
```

Comment:
It's for fixture purpose here. I want to have some committees to display in the results area. Obviously this repository method will be updated.